packages/integration-sdk-runtime/src/execution/__tests__/jobState.test.ts

```diff
@@ -0,0 +1,62 @@
+import {
```

Comment:
Bonus: Added unit tests for `jobState.ts`

Reply:
@austinkelleher stop flexing